terraform/sponsored_iam.tf

```diff
@@ -1,10 +1,15 @@
+resource "aws_iam_group" "sponsored_writers" {
+  provider = aws.sponsored
+
+  name = "writers"
+}
 
-resource "aws_iam_policy" "sponsored_writers" {
+resource "aws_iam_group_policy" "sponsored_writers" {
```

Comment:
Note, this retains the `writers` group (since the non-TF-managed `satra` user is still a member of it). However, it simplifies the policy by directly adding it inline to the group, instead of attaching it via standalone customer-managed policy.